2019-01-07  Aurelien Jarno  <aurelien@aurel32.net>

	[BZ #24046]
	* localedata/locales/en_US (date_fmt): Add, set to
	"%a %d %b %Y %r %Z".

--- a/localedata/locales/en_US
+++ b/localedata/locales/en_US
@@ -117,6 +117,9 @@ t_fmt   "%r"
 % Appropriate AM/PM time representation (%r)
 t_fmt_ampm "%I:%M:%S %p"
 %
+% Appropriate date and time representation for date(1)
+date_fmt "%a %d %b %Y %r %Z"
+%
 % Strings for AM/PM
 %
 am_pm	"AM";"PM"
